### PR TITLE
Certificate tutorials

### PR DESCRIPTION
**Pull request by borditamas**

### CODE REVIEW

**Review by MaGaMeGa — APPROVED** [review]
create cloud cert - typo - in 9th Step selest - select

**Review by rbocsi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -356,8 +356,33 @@ New payload - you can easily map the old fields to the new ones.
...
+The Key-Store is intended to store the certificates and/or key-pair certificates. Key-pair certificates are contain the certificate chain with some additinal data, such as the private-public keys, which are necessary for the secure operation. Certificates located in this store (without the keys) will be attached to the outcoming HTTPS requests. Arrowhead Framework is designed for handling the `p12` type of Key-Stores.
+
+*(**Note:** When you creating a new key-pair certificate, then the `key-password` and the `key-store-password` must be the same.)*
+
+### The Trust-Store
+
+The Trust-Store is containing those certificates, what the web-server considers as trusted ones. Arrowhead Framework is designed for handling the `p12` type of Trust-Stores. Typically your Trust-Store should contain only the cloud certificate, which ensures that only those incoming HTTPS requests are authorized to access, which are having this certificate within their certificate chain.
+
+### How to create my own certificates?
+Currently Arrowhead communitiy have the possibility to create only "self signed" certifications. See the tutorials:
```
Comment (tsvetlin) [inline README.md]: Typo: Community

File: README.md [hunk 2]
```diff
@@ -4092,4 +4117,4 @@ __PriorityList__ is the input
 > **Note:** The keys of the map are Orcherstrator store rule IDs, the values are the new priorities.
 
 > **Note:** 4.1.2 version: PUT /orchestrator/mgmt/store/priorities<br />
-            Same as the new version
```
Comment (tsvetlin) [inline README.md:4095]: Why is this change needed?

**Review by tsvetlin — CHANGES_REQUESTED** [review]
Fix the typo, and it can be merged.

File: README.md (hunk 2 above)
Comment (borditamas) [reply README.md:4095]: Have no idea..., but will discard this change

Comment (borditamas) [reply README.md:4095]: Actually, there is no change at all... don't know why Github highlight it....

**Pull request merged**